### PR TITLE
support CamelCase for param names in routers

### PR DESCRIPTION
**Pull request by matemax**
Add support CamelCase for  param names in routers

### CODE REVIEW

File: tests/test_routing.py [hunk 1]
```diff
@@ -140,3 +140,31 @@ def test_conditional_check_proper_compile(handler):
     router.finalize()
 
     assert router.finalized
+
+
+@pytest.mark.parametrize(
+    "param_name",
+    (
+        "fooBar", "foo_bar",
```
Comment (ahopkins) [inline tests/test_routing.py:148]: Can you add `Foobar` and `foobar1` as tests? In theory any valid python variable should work here.

**Review by ahopkins — CHANGES_REQUESTED** [review]
Thanks

File: tests/test_routing.py (hunk 1 above)
Comment (ahopkins) [reply tests/test_routing.py:148]:
```
([a-zA-Z_][a-zA-Z0-9_]+)
```

Comment (matemax) [reply tests/test_routing.py]: Add a support `foobar1` and `Foobar`

**Review by ahopkins — APPROVED** [review]

**Pull request merged**